tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1927,7 +1927,117 @@ def test_large_timestamp_generation(self):
 
         generated_ids = model.generate(input_features, max_length=448, return_timestamps=True).to("cpu")
 
-        EXPECTED_OUTPUT = torch.tensor([50258, 50259, 50360, 50365, 2221, 13, 2326, 388, 391, 307, 264, 50244, 295, 264, 2808, 5359, 11, 293, 321, 366, 5404, 281, 2928, 702, 14943, 13, 50629, 50682, 6966, 307, 2221, 13, 2326, 388, 391, 311, 9060, 1570, 1880, 813, 702,  1871, 13, 50870, 50911, 634, 5112, 505, 300, 412, 341, 42729, 3196, 295, 264,  1064,  11, 365,  5272,   293, 12904,  9256, 450, 10539, 949, 505, 11, 51245, 51287,  1034, 4680, 10117, 490, 3936, 293, 1080,  3542, 5160, 881, 26336, 281, 264, 1575, 13, 51494, 51523, 634, 575, 12525, 22618, 1968,  6144, 35617, 1456, 397, 266, 311, 589, 307, 534, 10281, 934, 439, 11, 51799, 51815, 50257])
+        EXPECTED_OUTPUT = torch.tensor(
```

Comment:
I don't think we want to split across the lines like this. 

You can wrap `EXPECTED_OUTPUT` around `# fmt: off` and `fmt: on` comments to avoid this

Reply:
Ok thanks for the tips! will be useful ;)

Reply:
You can also use `# fmt: skip` for single lines, _c.f._ the previous comment https://github.com/huggingface/transformers/pull/30812#discussion_r1603595554